functime/feature_extraction/tsfresh.py

```diff
@@ -424,13 +428,13 @@ def cid_ce(x: TIME_SERIES_T, normalize: bool = False) -> FLOAT_EXPR:
     float | Expr
     """
     if normalize:
-        y = (x - x.mean()) / x.std()
+        y = (x - x.mean()) / x.std(ddof=0)
     else:
         y = x
 
     if isinstance(x, pl.Series):
-        diff = np.diff(x)
-        return np.sqrt(np.dot(diff, diff))
+        diff = y - y.shift(periods=-1)
```

Comment:
Using diff is faster because y.shift creates another Series, while diff just takes two values at a time... 

Using .dot is faster because there is no intermediate Series being created. It is just accumulation on a single value.

Reply:
I believe we agreed that it is ok to use NumPy for eager case. Let's revert this too.

Reply:
Yes, I know, I was busy on something else.